src/source/PeerConnection/PeerConnection.c

```diff
@@ -883,6 +889,32 @@ STATUS peerConnectionOnConnectionStateChange(PRtcPeerConnection pRtcPeerConnecti
     return retStatus;
 }
 
+STATUS peerConnectionOnSenderBandwidthEstimation(PRtcPeerConnection pRtcPeerConnection, UINT64 customData,
+                                                 RtcOnSenderBandwidthEstimation rtcOnSenderBandwidthEstimation)
+{
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
+    PKvsPeerConnection pKvsPeerConnection = (PKvsPeerConnection) pRtcPeerConnection;
+    BOOL locked = FALSE;
+
+    CHK(pKvsPeerConnection != NULL && rtcOnSenderBandwidthEstimation != NULL, STATUS_NULL_ARG);
+
+    MUTEX_LOCK(pKvsPeerConnection->peerConnectionObjLock);
+    locked = TRUE;
+
+    pKvsPeerConnection->onBandwidth = rtcOnSenderBandwidthEstimation;
```

Comment:
This is vague. To be more descriptive, can we change `onBandwidth` to `onSenderBandwidthEstimation`?